src/app/utils/helper.ts

```diff
@@ -0,0 +1,235 @@
+import postcss from 'postcss';
+import cssjs, { CssInJs } from 'postcss-js';
+import sectionMockJson from '../mocks/sections.json';
+import { MockDocumentDataType } from '../types/document.type';
+
+const allKeysToReplace = [
```

Comment:
These will be dynamic.
Can the keys be generated by the values in the section config `headerFields` and `bodyFields`?
This also assumes we every field name across the document would be unique (which is not true). We should be able to reuse `title` in different sections for example.
Perhaps this should be an object based on a sections ID and then keys to replace?

Reply:
Fixed.